php/libraries/File_Upload.class.inc

```diff
@@ -349,7 +349,17 @@ class File_Upload
             return false;
         }
         //Test each method
-        foreach ($this->fileHandlers[$field]['methods'] AS $method) {
+        $methods = $this->fileHAndlers[$field]['methods'];
+        if (is_iterable($methods)) {
+            // Phan thinks methods is a string and fails below without
```

Comment:
I don't think that would work, because it's not a class variable being iterated over, it's an item inside of the class variable that's being iterated over.